src/components/input/Input.ts

```diff
@@ -0,0 +1,24 @@
+import Block from '../../utils/Block';
+import template from './template.hbs';
+import './style.less';
+
+export interface InputProps {
+    classWrapper: string; //'input' || 'text'
+    type: string;
+    name: string;
+    labelValue: string;
+    classes?: string;
+    placeholder?: string;
+    errorValue?: string;
+    disabled?: string;
```

Comment:
по названию сразу понятно, что это булева переменная. 
```suggestion
    disabled?: boolean;
```